Tests/Mapping/PropertyMappingTest.php

```diff
@@ -33,52 +33,10 @@ public function testConfiguredMappingAccess()
         $this->assertEquals('fileName', $prop->getFileNamePropertyName());
     }
 
-    /**
```

Comment:
why removed?

Reply:
Because enforcing parameter type is making tests failing. Of course, if you know a better way, I'm open to suggestions